scripts/install/linux/install-linux.sh

```
@@ -25,6 +25,20 @@ BLUE='\033[0;34m'
 CYAN='\033[0;36m'
 NC='\033[0m'
 
+# Installation paths
+readonly FNM_INSTALL_DIR="${FNM_DIR:-$HOME/.local/share/fnm}"
+readonly FNM_BIN_DIR="$FNM_INSTALL_DIR"
+readonly INSTALLATION_LOG="/tmp/talawa-install-$$.log"
```

Shellcheck (0.11.0)

[warning] INSTALLATION_LOG appears unused. Verify use (or export if used externally).

(SC2034)